Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/CosmosServerlessSparkBatchJob.java

```diff
@@ -253,10 +253,18 @@ public String getJobUuid() {
                                                            int batchId,
                                                            int startIndex,
                                                            int maxLinePerGet) {
-        String requestUrl = String.format("%s/%d/log?from=%d&size=%d", livyUrl, batchId, startIndex, maxLinePerGet);
-        return getHttp()
-                .withUuidUserAgent()
-                .get(requestUrl, null, null, SparkJobLog.class);
+        String requestUrl = String.format("%s/batches/%d/log", livyUrl, batchId);
+        List<NameValuePair> parameters = ImmutableList.of(
+                new BasicNameValuePair("from", String.valueOf(startIndex)),
+                new BasicNameValuePair("size", String.valueOf(maxLinePerGet)));
+        return new HttpObservable() {
+            @Override
+            public List<NameValuePair> getDefaultParameters() {
+                // default parameters for class HttpObservable is "apiVersion=$apiVersion",
+                // which is unnecessary and will make the request URL strange.
```

Comment:
`apiVersion` parameter is unnecessary?

Reply:
`apiVersion` is required since the request is sent to Azure service, which is different from submission log request for HDInsight spark job.